llvm/lib/SYCLLowerIR/ESIMD/LowerESIMD.cpp

```diff
@@ -805,8 +811,8 @@ static APInt parseTemplateArg(id::FunctionEncoding *FE, unsigned int N,
     const std::string_view &TyStr = ValL->getType();
     if (Conv == ESIMDIntrinDesc::GenXArgConversion::NONE && TyStr.size() != 0)
       // Overwrite Ty with IntegerLiteral's size
-      Ty =
-          parsePrimitiveTypeString(StringRef(&*TyStr.begin(), TyStr.size()), Ctx);
+      Ty = parsePrimitiveTypeString(StringRef(&*TyStr.begin(), TyStr.size()),
```

Comment:
It is rather exception here.
clang-format produced lots of good changes to lines modified by me and only 1 or 2 changes unrelated to the main PR. In this particular case I decided to keep those 1-or-2 unrelated clang-format fixes instead of searching for it and cancelling.

Normally, if there are many unrelated clang-format changes, I cancel them an apply only required for modified lines.

Reply:
@v-klochkov, btw, do you apply clang-format to the whole file, or to the diff? If applied to the diff, these issues will not arise. Can share a command if needed.